#187 Update UIOverview page v1.6.0 (#196)
Inbibo/adonisfx_docs: docs/maya/ui_overview.md

#### Utils

- **Upgrade Deprecated Nodes**. Refreshes the node graph and upgrades all deprecated nodes making sure that the scene still works the same.
- **Clear**. Removes all AdonisFX nodes from the scene.

<br/>

> - **Mirror**. Opens the Mirror Tool UI to mirror the muscle setup from one side of the body to the other.

Reviewer: ```suggestion
- **Mirror**. Opens the Mirror Tool UI which allows to mirror the muscle setup (locators, sensors and muscles) from one side of the character to the other based on left and right naming rules.
```

Author: Applied.

- **Paint Tool**. Opens the Paint Tool UI to modify the paintable maps in AdnSkin, AdnMuscle and AdnRibbonMuscle deformers. This item corresponds to the shelf button ![Paint Tool](images/adn_paint_tool.png){style="width:4%"}.
- **Interactive Playback**. Enables Maya Interactive Playback. In this playback mode, dynamic interaction with objects and parameters while simulating is allowed. This item corresponds to the shelf button ![Interactive Playback](images/adn_interactive_playback.png){style="width:4%"}.

### I/O section

> - **Import (beta)**. Opens the Import UI which allows to import an AdonisFX rig into the scene.

Reviewer: ```suggestion
- **Import (beta)**. Opens the Import UI which allows to import an AdonisFX rig from a file in disk into the scene.
```

Author: Done.

> - **Export (beta)**. Opens the Export UI which allows to export an AdonisFX rig from the scene.

Reviewer: ```suggestion
- **Export (beta)**. Opens the Export UI which allows to export an AdonisFX rig from the scene into a file in disk.
```

Author: Done.
